### ISSUE TITLE
Message delivery failed

### ISSUE THREAD

**Issue by edsonblwolf** [odoo/odoo#75032]
Impacted versions: 12.0

Steps to reproduce: Redefine password

Current behavior: ('Message delivery failed', "Email sending failed by SMTP server'None'.\nSMTPSenderRefused: 550\n5.7.1Unauthorized shipping - Check your domain's MX and/or SPF\nbounce+12-res.users-2@loja.xxx.br") 

Expected behavior: Send password to email

Video/Screenshot link (optional): 

Support ticket number submitted via odoo.com/help (optional):

The system is set up right, it is sending invoices normally.

**Comment by pedrobaeza** [comment]
This is a bad configuration of your outgoing mail server in your end.

Closing as no Odoo issue.

**Comment by edsonblwolf** [comment]
But it's sending out invoices to customers, usually.

**Comment by pedrobaeza** [comment]
Well, not for that address. It seems no plus addressing or proper alias for the bounce one.

**Comment by edsonblwolf** [comment]
The email I changed to post here, but it's what's configured in general SMTP.

**Comment by pedrobaeza** [comment]
Sorry, for further questions, go to forums.

**Comment by edsonblwolf** [comment]
You're right it's a problem with the domain itself, I'm using it for testing and I still don't have the store domain. It should use this way when it is external and internal it does not do this validation. Thanks ...
